### PR TITLE
Update pinterest.js

### PR DESCRIPTION
**Pull request by creatoras**
Fix lookup og:image for missing and wrong mediaUrl.

### CODE REVIEW

File: src/js/services/pinterest.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = function(shariff) {
         title += ' - ' + creator;
     }
     var img = shariff.getOption('mediaUrl');
-    if (img && img.length <= 0) {
+    if (!img || img.length <= 7) {
```
Comment (compeak) [inline src/js/services/pinterest.js]: Shouldn't this be at least 10 characters? I think `http://a.b` makes more sense as a minimum URL than `http://`.

Comment (creatoras) [inline src/js/services/pinterest.js]: Yes. I'm also fine with < 11 or 13 if we say it should be a jpg or png or so. It was just a fast fix for a System with wrong output of null.

**Comment by craiq** [comment]
For what you check the length over 0? For link validation?
Just the case the link has the failure being to short? Whats with all the other possibilities?
When you really want to be sure the file exists you have to do a ajax request and check it.
Or you just give the possibility to enter a wrong link, being to short or whatever, and let it as it was.

Some other link validation would be http://stackoverflow.com/questions/2723140/validating-url-with-jquery-without-the-validate-plugin but still the file must not exist, what wont make a difference to a wrong link at the end.

**Comment by compeak** [comment]
We're not doing validation checks in any other service. We only check if something is there, not whether it is broken. So out of consistency this will not be merged.

**Comment by JPustkuchen** [comment]
The code is wrong anyway. You should check for null furthermore to use og:image instead. Currently this doesn't work if "img" is null:

if (img && img.length <= 0) {

should be

if ((img && img.length <= 0) || img === null) {

**Pull request closed without merging**